ironfish/src/utils/yup.ts

```diff
@@ -83,7 +79,7 @@ export class YupUtils {
     value: unknown,
     options?: yup.ValidateOptions<unknown>,
   ):
-    | { result: YupSchemaResultSync<S>; error: null }
+    | { result: YupSchemaResult<S>; error: null }
```

Comment:
`YupSchemaResult` is already synchronous but oddly `YupSchemaResultSync` was not.